tasks/bootstrap-packages.yml

```diff
@@ -7,6 +7,5 @@
 
 - name: Install extra packages
   apt:
-    name: "{{ item }}"
+    name: "{{ sb_debian_base_extra_packages | union(sb_debian_base_supplementary_packages) }}"
```

Comment:
@speedlight thank you for pushing this forward. This looks like a backward-incompatible change, which might be OK if we drop support for older Ansible versions, however, I think we still need to bump the major version and update the role `min-_ansible_version` before release.

Reply:
Thanks for pointing this out, I think we can now use ansible 2.7 and prepare another PR to move to 2.8, currently 2.7 is in `Maintained (security fixes)` state so we have some window to drop that version. WDYT?